csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -4339,6 +4341,38 @@ The relevant path for this test is:
 > There is no action statement for `CSAFPID-9080702`.
 > Note: The action statement for `CSAFPID-9080700` and `CSAFPID-9080701` is given through `CSAFGID-0001`.
 
+#### 6.1.27.11 Vulnerabilities
+
+It MUST be tested that the element `/vulnerabilities` exists.
```

Comment:
Very good. Explicit often is better than implicit - less discussions troubleshooting interoperability